### PR TITLE
Use _default variables in heartbeat RAs.

### PR DESCRIPTION
**Pull request by sebastic**
Triggered by the review of #1340, the existings RAs have been updated to also use `_default` variables to make them more in line with current best practices.

### CODE REVIEW

**Comment by oalbrigt** [comment]
Nice.

Due to the number of agents it affects I'll review this and merge after the current release schedule to ensure it gets some months worth of testing by the upstream community before making it in a release.

**Comment by sebastic** [comment]
Sure, no problem. I'll try to remember to rebase it periodically in the mean time.

File: heartbeat/NodeUtilization [hunk 1]
```diff
@@ -32,6 +32,26 @@
 : ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
 
+# Parameter defaults
+
+OCF_RESKEY_pidfile_default="$HA_VARRUN/NodeUtilization-${OCF_RESOURCE_INSTANCE}"
```
Comment (oalbrigt) [inline heartbeat/NodeUtilization]: I dont see any pidfile parameter in this agent.

Comment (oalbrigt) [reply heartbeat/NodeUtilization]: Nevermind. Monday morning brain talking :)

**Comment by oalbrigt** [comment]
LGTM. Thanks.

I also did a check of the metadata before and after, and it only changed for like 3 parameters, where it was showing the incorrect value before.

**Pull request merged**